src/main/resources/static/surveyor.js

```diff
@@ -0,0 +1,52 @@
+$(document).ready(function() {
+    //Add's new multiple choice question
+    $(".new-MCQ").click(function(){
+        var numOptions = Math.floor($("#MCQOptionsNum").val());
+        var newMCQ = `<div class='multipleChoiceQuestion' id="question">
+                        <label for='QuestionText'>Question:</label>
+                        <input type='text' id='QuestionText'>`;
+        if(numOptions < 1){
+            alert("Number of Multiple Choice Options must be greater than 0!");
```

Comment:
an MCQ usually has minimum two options (true, false)

Reply:
Should be "greater than 1!" in the error message

Reply:
updated it.